src/components/challenges/active.tsx

```diff
@@ -2,6 +2,9 @@ import { FC, useState } from 'react';
 import Card from '@/components/Card/index';
 import details from '@/components/challenges/details';
 import participantsDetails from '@/components/challenges/participants';
+import { toast } from 'react-toastify';
+
+const SUBSCRIBE_TO_CHALLENGE_URL = `${process.env.NEXT_PUBLIC_BASE_URL}/challenges/subscribe`;
```

Comment:
I think it will be good if we move this to an AppConstants file that holds all our application constants.
cc - @whyDontI

Reply:
@akshay1502 Please create an `urls.js` file in the `components/constants` directory and move these variables there. 👍🏽